modules/sdk-coin-atom/src/lib/constants.ts

```diff
@@ -1,12 +1,5 @@
-export const DEFAULT_SEED_SIZE_BYTES = 16;
-export const sendMsgTypeUrl = '/cosmos.bank.v1beta1.MsgSend';
-export const delegateMsgTypeUrl = '/cosmos.staking.v1beta1.MsgDelegate';
-export const undelegateMsgTypeUrl = '/cosmos.staking.v1beta1.MsgUndelegate';
-export const withdrawDelegatorRewardMsgTypeUrl = '/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward';
 export const validDenoms = ['natom', 'uatom', 'matom', 'atom'];
 export const accountAddressRegex = /^(cosmos)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
 export const validatorAddressRegex = /^(cosmosvaloper)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
-export const contractAddressRegex = /^(cosmos)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']+)$/;
```

Comment:
why removing contractAddressRegex ?

Reply:
Contract support was only added in abstract-cosmos & other chains but wasn't added for Atom it seems, so this was unused. I've updated the PR with the contract support changes needed for atom as well.